site/layouts/shortcodes/orange-footer.html

```diff
@@ -59,92 +59,90 @@ <h3 class="footer-heading">Follow us</h3>
 {{- end }}
 {{- end }}
 {{- if $nav }}
-  <div class="container-xxl footer-nav">
-    <nav class="accordion accordion-dark" id="accordion{{ $id }}" aria-label="Sitemap footer {{ $id }}">
-      <div class="row">
-        <div class="footer-column col-md-3">
-          <h3 class="accordion-header footer-heading" id="headingOne{{ $id }}">
-            <button class="accordion-button collapsed container-xxl px-1 d-md-none" type="button" data-bs-toggle="collapse" data-bs-target="#collapseOne{{ $id }}" aria-expanded="true" aria-controls="collapseOne{{ $id }}">
-              Category
-            </button>
-            <span class="d-none d-md-flex">Category</span>
-          </h3>
-          <div id="collapseOne{{ $id }}" class="container-xxl accordion-collapse collapse" data-bs-parent="#accordion{{ $id }}">
-            <ul class="navbar-nav">
-              <li><a class="nav-link" href="#" aria-describedby="headingOne{{ $id }}">Subcategory</a></li>
-              <li><a class="nav-link" href="#" aria-describedby="headingOne{{ $id }}">Subcategory</a></li>
-              <li><a class="nav-link" href="#" aria-describedby="headingOne{{ $id }}">Subcategory</a></li>
-              <li><a class="nav-link" href="#" aria-describedby="headingOne{{ $id }}">Subcategory</a></li>
-              <li><a class="nav-link" href="#" aria-describedby="headingOne{{ $id }}">Subcategory</a></li>
-              <li><a class="nav-link" href="#" aria-describedby="headingOne{{ $id }}">Subcategory</a></li>
-              <li><a class="nav-link" href="#" aria-describedby="headingOne{{ $id }}">Subcategory</a></li>
-              <li><a class="nav-link" href="#" aria-describedby="headingOne{{ $id }}">Subcategory</a></li>
-            </ul>
-          </div>
+  <nav class="accordion accordion-dark container-xxl footer-nav" id="accordion{{ $id }}" aria-label="Sitemap footer {{ $id }}">
```

Comment:
Not 100% sure about this change, need to see if there is another way to handle this